projects/scripts/adi_board.tcl

```diff
@@ -717,7 +717,7 @@ proc ad_cpu_interrupt {p_ps_index p_mb_index p_name} {
 
   if {($sys_zynq == 2) && ($p_index <= 7)} {
```

Comment:
I think we should do this in all three use cases.

Reply:
Yes, I forgot about the other two.

Reply:
I will squash everything into one commit after a test.